impl/client/src/main/javascript/web/pentaho/visual/config/vizApi.conf.js

```diff
@@ -369,13 +369,23 @@ define(function() {
           extension: {
             // Plot
             sizeAxisRatio: 1 / 5,
-            sizeAxisRatioTo: "height", // plot area client height
+            sizeAxisRatioTo: "minwidthheight",
             sizeAxisOriginIsZero: true,
 
             // . dot
+            dot_lineWidth: function() {
+              // show nullShape
+              var v = !this.panel.visualRoles.size.isBound() || this.getSize() != null ? 0 : this.delegate();
+              return this.finished(v);
+            },
+            dot_strokeStyle: fillStyle2,
             dot_shapeSize: function() {
               var v = this.panel.visualRoles.size.isBound() ? this.delegate() : (5 * 5);
               return this.finished(v);
+            },
+
+            dot_DefaultVisible: function() {
```

Comment:
No, just something I forgot to delete. Was trying to show the null bubble even when there is no color bound (https://github.com/webdetails/ccc/blob/master/package-res/ccc/core/base/sign/dotSizeColor-sign.js#L53-L56).